test/unit/create-contentful-api-test.js

```diff
@@ -352,3 +352,65 @@ test('CDA call sync fails', (t) => {
     teardown()
   })
 })
+
+test('Given json should be parsed correctly as a collection of entries', (t) => {
+  const client = contentful.createClient({accessToken: 'blablabla', space: 'bla'})
```

Comment:
I think we shouldn't do it like that here.

all the other test cases use something like

```
const {api} = setupWithData({
    promise: Promise.resolve({ data: contentTypeMock })
})
entitiesMock.contentType.wrapContentType.returns(contentTypeMock)

return api.getContentType('ctid')
```

for the matter of consistency and to have on less file to load ( 'require( '../../' )? ) it might make sense to use the same approach here.